locale/en/get-involved/node-events.md

```diff
@@ -10,11 +10,15 @@ This is a list of Node.js events and conferences. Please submit a PR if you'd li
 ## Notes for adding events
 
 FORMAT
-- [Event Name](https://) - Location, Dates
+
+- [Event Name](https://) | Location | Dates
```

Comment:
Since you are here, could you please also change the `Event Name` to normal text, the link `https://` seems invalid here.

```suggestion
- Event Name | Location | Dates
```

Reply:
The idea is that the event is actually a link to the website, look at the example and the contents I added

Reply:
Seems reasonable. 

Then how about just remove `https://` in the scope ?

Now on our website [Node.js event](https://nodejs.org/en/get-involved/node-events/) page, clicking here(maybe accidentally) would lead to an empty page.

Reply:
True, that could be confusing. I'll remove just that reference.

Reply:
IMO：Here the 'Event Name' is ONLY a tip for you to give a link as a tip, so I don't think we need to exclipit show the link. I think those submitters will add the links for their event names as what others do,

Reply:
What about using a pure text plain instead? Because others will see they should use link, or just use an empty link instead (don't need to open up a new window).... :)

Reply:
Agree!